src/cmd.js

```diff
@@ -101,6 +92,7 @@ function _cmd(options, command, commandArgs, userOptions) {
   var requiredOptions = {
     input: pipe,
     shell: false,
+    stripFinalNewline: false,
```

Comment:
Can you move this a few lines up and put this inside of `defaultOptions`?

Reply:
done